drivers/flash/soc_flash_nrf.c

```diff
@@ -384,6 +382,9 @@ static int work_in_time_slice(struct flash_op_desc *p_flash_op_desc)
 	if (err != TICKER_STATUS_SUCCESS && err != TICKER_STATUS_BUSY) {
 		result = -ECANCELED;
 	} else if (k_sem_take(&sem_sync, K_MSEC(FLASH_TIMEOUT_MS)) != 0) {
+		/* Stop any scheduled jobs */
+		_ticker_stop(instance_index, 0, ticker_id);
```

Comment:
use user_id value of 3, as the call is done from thread context.

Reply:
OK. Changed. Seems that stop does not care, since 0 also worked. Wes should work out some names for the UIDs... 3 is rather cryptic.